Makefile

```diff
@@ -155,6 +160,8 @@ verify:
 	$(MAKE) verify-manifests
 	echo "Checking for unsynced nested [go.mod,go.sum] files"
 	$(MAKE) verify-nested-vendor
+	echo "Checking commit integrity"
+	$(MAKE) verify-commits
```

Comment:
just following the convention here, but is there a reason we don't just tack these on to `verify` as prereqs?